Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -272,15 +269,20 @@ define([
 
         var batchTableResources = new Cesium3DTileBatchTableResources(this, instancesLength);
         this.batchTableResources = batchTableResources;
-        if (batchTableJSONByteLength > 0) {
-            var batchTableString = getStringFromTypedArray(uint8Array, byteOffset, batchTableJSONByteLength);
-            batchTableResources.batchTable = JSON.parse(batchTableString);
-            byteOffset += batchTableJSONByteLength;
+        if (batchTableJsonByteLength > 0) {
```

Comment:
Personally I prefer it how it is since it keeps tile parsing all in the same place, and interacts easily with the `byteOffset` and other things.